Library/Formula/gtk-murrine-engine.rb

```diff
@@ -17,4 +16,9 @@ def install
                           "--enable-animation"
     system "make", "install"
   end
+
+  test do
+    assert (lib/"gtk-2.0/2.10.0/engines/libmurrine.so").exist?
+    assert (share/"gtk-engines/murrine.xml").exist?
+  end
```

Comment:
this is another test that just tests that files are installed, can we remove it?